alephium/contracts/token_bridge/token_bridge.ral

```diff
@@ -117,7 +117,7 @@ Contract TokenBridge(
         assert!(size!(localTokenId) == 32, ErrorCodes.InvalidTokenId)
         assert!(size!(nonce) == 4, ErrorCodes.InvalidNonceSize)
         if (localTokenId != wrappedAlphId) {
-            assert!(tokenRemaining!(payer, localTokenId) == 1, ErrorCodes.TokenNotExist) // make sure that the token exists
```

Comment:
I realized that it might be hard to specify 1 coin if the coin amount is in `float`.